docs/development/core/development-functional-tests.asciidoc

```diff
@@ -61,6 +61,20 @@ export TEST_ES_PASS=<password>
 node scripts/functional_test_runner
 ----------
 
+** Selenium tests can be run in headless mode by setting the environment variable below.  Unset this variable to show the browser.
++
+["source", "shell"]
+----------
+export TEST_BROWSER_HEADLESS=1
+----------
+
+** When running against cloud deployment, some tests are not applicable use --exclude-tag to skip those tests.  An example shell file can be found at:  {blob}test/scripts/jenkins_kibana.sh[test/scripts/jenkins_kibana.sh]
```

Comment:
maybe new sentence ?  ` some tests are not applicable.  Use --exlude-tag..`  feel free to ignore